client/testutil/driver_compatible.go

```diff
@@ -18,6 +19,15 @@ func QemuCompatible(t *testing.T) {
 	}
 }
 
+func RktCompatible(t *testing.T) bool {
+        if runtime.GOOS == "windows" || syscall.Geteuid() != 0 {
+                t.Skip("Must be root on non-windows environments to run test")
+        }
+        // else see if rkt exists
+        _, err := exec.Command("rkt", "version").CombinedOutput()
+        return err == nil
```

Comment:
Shouldn't we skip if rkt isn't installed or can't be executed? There is no test for this bool anywhere.